src/pages/TagCreator/Organization.js

```diff
@@ -134,15 +135,15 @@ export const OrgChange = ({ orgName, setOrgTags, changeValue, setDisplayState })
   const handleSubmitOrg = () => {
     const topics = []
     if (orgName) {
-      axios.get(`${process.env.REACT_APP_API_URL}/api/organizations/` + orgName,)
+      const og = orgName.replace(/ /g,"-").toLowerCase()
+      axios.get(`${process.env.REACT_APP_API_URL}/api/organizations/`+og,)
```

Comment:
Does the following axios request format work:

axios.get(`${process.env.REACT_APP_API_URL}/api/organizations/${og}`)

Reply:
changed as suggested.